src/main/java/com/hubspot/jinjava/interpret/UnknownTokenException.java

```diff
@@ -5,8 +5,8 @@
   private static final long serialVersionUID = -388757722051666198L;
   private final String token;
 
-  public UnknownTokenException(String token, int lineNumber, int startPosition) {
-    super("Unknown token found: " + token, lineNumber, startPosition);
+  public UnknownTokenException(String token) {
```

Comment:
we definitely want to keep these line numbers and positions in the error message.

Reply:
since the exception is being thrown at a completely new place now, there seems no way to know the line number and the starting position of the token, hence I changes the method signature.

Let me see if I can make some changes to alter this back to exactly how it was :+1:

Reply:
<img width="1065" alt="Screenshot 2019-06-26 at 11 50 02 AM" src="https://user-images.githubusercontent.com/25414121/60155749-921c4f80-9808-11e9-8d03-996adcfeb9c7.png">

currently, it doesn't show any lineNumber or startingPosition as well. So I guess removing it is okay?
The above screenshot was taken after I created a test and ran it on the current code in the master branch and it seems not to show the lineNumber and the startingPosition of the variable that is not found.

Reply:
It's not in the `toString()` method, but we use it to build errors when we catch these exceptions.